packages/desktop-client/src/hooks/useUndo.ts

```diff
@@ -26,19 +29,21 @@ export function useUndo(): UndoActions {
 
   const showUndoNotification = useCallback(
     (notification: Notification) => {
-      dispatch(
-        addNotification({
-          type: 'message',
-          timeout,
-          button: {
-            title: 'Undo',
-            action: dispatchUndo,
-          },
-          ...notification,
-        }),
-      );
+      if (isNarrowWidth) {
+        dispatch(
+          addNotification({
+            type: 'message',
+            timeout,
+            button: {
+              title: 'Undo',
+              action: dispatchUndo,
+            },
+            ...notification,
+          }),
+        );
+      }
```

Comment:
Nit: return early instead to save us an indentation. Also, I think we should do the same on `showRedoNotification` function below.
```suggestion
      if (!isNarrowWidth) {
        return;
      }

      dispatch(
        addNotification({
          type: 'message',
          timeout,
          button: {
            title: 'Undo',
            action: dispatchUndo,
          },
          ...notification,
        }),
      );
```

Reply:
Both changes committed.